images/x86_64/amazonlinux2022/5.15/Dockerfile

```diff
@@ -25,8 +25,6 @@ RUN dnf groupinstall -y 'Development Tools' && \
 		xargs -I@ curl -Lo headers.rpm ${URL}@ && \
     grep -E 'kernel-devel-'${VERSION}'.*.rpm' primary.xml | grep href | cut -d\"  -f2 | \
 		xargs -I@ curl -Lo sources.rpm ${URL}@ && \
-    grep -E 'bpftool-'${VERSION}'.*.rpm' primary.xml | grep href | cut -d\"  -f2 | \
```

Comment:
`bpftool` is not needed since scap-open with `modern-bpf` built in gets built in a specific step with a specific rootfs.